documentation/modules/metrics/con-metrics-cruise-control.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// metrics/assembly-metrics.adoc
+
+[id='con-metrics-cruise-control-{context}']
+= Monitoring Cruise Control operations
+
+[role="_abstract"]
+Kafka metrics are gathered by Cruise Control in order to track the utilization of brokers, topics, and partitions.
+Cruise Control also provides a set of metrics for monitoring its own performance.
+
+The Cruise Control metrics reporter collects raw metrics data from Kafka brokers.
+The data is produced to topics that are automatically created by Cruise Control.
+The metrics are used to link:{BookURLUsing}#proc-generating-optimization-proposals-str[generate optimization proposals for Kafka clusters^].
+
+Cruise Control metrics are available for real-time monitoring of Cruise Control operations.
+For example, you can use Cruise Control metrics to monitor the status of rebalancing operations that are running or provide alerts on any anomalies that are detected in an operation's performance.
+
+You can expose Cruise Control metrics by enabling JMX for Cruise Control.
+For a full list of available Cruise Control metrics, which are known as _sensors_, see the {CruiseControlSensorsDocs}.
+
+== Exposing Cruise Control metrics
+
+If you want to expose metrics on Cruise Control operations, configure the `Kafka` resource xref:proc-metrics-kafka-deploy-options-{context}[to deploy Cruise Control metrics configuration].
+You can use your own configuration or use the example `kafka-cruise-control-metrics.yaml` file provided by Strimzi.
+
+You add the configuration to the `metricsConfig` of the `CruiseControl` property in the `Kafka` resource.
+The configuration enables JMX for Cruise Control and uses the {JMXExporter} to expose Cruise Control metrics through an HTTP endpoint.
```

Comment:
Thanks Jakub, I misunderstood this but you are right, we don't expose JMX since we never set `KAFKA_JMX_OPTS` [1] for the Cruise Control container. We only enable the Prometheus JMX Exporter [2] which "exposes Cruise Control metrics through an HTTP endpoint"

[1] https://github.com/linkedin/cruise-control/blob/migrate_to_kafka_2_4/kafka-cruise-control-start.sh#L65
[2] https://github.com/strimzi/strimzi-kafka-operator/blob/main/docker-images/kafka/cruise-control-scripts/cruise_control_run.sh#L28

Reply:
Changed to: _The configuration enables the {JMXExporter} to expose Cruise Control metrics through an HTTP endpoint._